src/adapter/graphql/src/root.rs

```diff
@@ -91,6 +91,22 @@ impl Mutation {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct AccessToken {
+    pub token: String,
+}
+
+impl AccessToken {
+    pub fn new<S>(token: S) -> Self
+    where
+        S: Into<String>,
+    {
+        Self {
+            token: token.into(),
+        }
+    }
+}
```

Comment:
Should this live in `scalars/` sub-module?

Reply:
Why? It's not going outside API